editor/project_manager/project_list.cpp

```diff
@@ -76,11 +80,47 @@ void ProjectListItemControl::_notification(int p_what) {
 		case NOTIFICATION_MOUSE_ENTER: {
 			is_hovering = true;
 			queue_redraw();
+			queue_accessibility_update();
 		} break;
 
 		case NOTIFICATION_MOUSE_EXIT: {
 			is_hovering = false;
 			queue_redraw();
+			queue_accessibility_update();
+		} break;
+
+		case NOTIFICATION_ACCESSIBILITY_UPDATE: {
+			RID ae = get_accessibility_element();
+			ERR_FAIL_COND(ae.is_null());
+
+			DisplayServer::get_singleton()->accessibility_update_set_role(ae, DisplayServer::AccessibilityRole::ROLE_LIST_ITEM);
```

Comment:
`ACCESSKIT_ROLE_LIST` and `ACCESSKIT_ROLE_LIST_ITEM` are meant for bulleted or numbered lists, such as the ones used in the New Project dialog to list rendering engines characteristics. When items can be focused, selected and/or activated such as here, you want to use `ACCESSKIT_ROLE_LIST_BOX` and `ACCESSKIT_ROLE_LIST_BOX_OPTION`.